pymc3/sampling.py

```diff
@@ -533,6 +535,9 @@ def sample(
 
     discard = tune if discard_tuned_samples else 0
     trace = trace[discard:]
+    trace.report._n_tune = tune
+    trace.report._n_draws = draws
```

Comment:
should this be like, `len(trace)`? I think if there's a KeyboardInterrupt, we'll still get a valid trace, it will just be short. This might also be true for `_n_tune` in the case of an interrupt.

Reply:
I updated it - see the other comment for screenshots on the KeyboardInterrupt behavior